src/app/app.component.scss

```diff
@@ -15,28 +15,58 @@
   background-color: $gray-f5;
 }
 
-// toolbar
-.title-bar {
+// navbar
+.navbar {
   width: 100vw;
   height: $toolbar-height;
-  background-color: $primary-color;
+  background-color: $artemis-dark;
   position:sticky;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 40px;
 }
 
-.title-bar-item-container {
-  padding: 10px 20px;
+.navbar-buttons {
```

Comment:
navbar-buttons and navbar-logo have exactly the same attributes. You could combine them in navbar-items (or similar)